### PR TITLE
chore: side panel cleanup

### LINKED ISSUE #903: UI: real estate wastage due to the left side panel
# Summary

We are wasting some real estate on the UI because we have a left-side panel that has no content. Let's remove that.

### PR DESCRIPTION
**Pull request by veds-g**
Fixes #903 

- Removed side panel on the left side
image

- Fixed warning with loaders added for pods view component, added a single loader for same
image

- Added error state
image

### CODE REVIEW

File: ui/src/App.tsx [hunk 1]
```diff
@@ -1,25 +1,18 @@
...
+import { Slide, ToastContainer } from "react-toastify";
+import { Routes, Route } from "react-router-dom";
 import { Breadcrumbs } from "./components/common/Breadcrumbs";
 import { Namespaces } from "./components/pages/Namespace";
 import { Pipeline } from "./components/pages/Pipeline";
 import logo from "./images/icon-on-blue-bg.png";
+
 import "./App.css";
-import { Slide, ToastContainer } from "react-toastify";
 import "react-toastify/dist/ReactToastify.css";
```
Comment (mshakira) [inline ui/src/App.tsx:14]: please fix the order of import

File: ui/src/utils/fetcherHooks/podsViewFetch.ts [hunk 2]
```diff
@@ -179,10 +179,21 @@ export const usePodsViewFetch = (
...
+    return true;
+  };
+
   //sets loading variable
   useEffect(() => {
-    if (pods && podsDetails) {
+    if (checkPodDetails()) {
       setLoading(false);
+    } else {
+      setLoading(true);
```
[the remark this answers is not in the document]
Comment (mshakira) [reply ui/src/utils/fetcherHooks/podsViewFetch.ts:196]: Rest of the code looks good.

[the remark this answers is not in the document]
Comment (veds-g) [reply ui/src/utils/fetcherHooks/podsViewFetch.ts:198]: added the error state

[the remark this answers is not in the document]
Comment (mshakira) [reply ui/src/utils/fetcherHooks/podsViewFetch.ts:198]: lgtm

**Review by mshakira — APPROVED** [review]

**Review by vigith — APPROVED** [review]

**Pull request merged**